pkg/asset/agent/image/baseiso.go

```diff
@@ -23,11 +23,9 @@ type BaseIso struct {
 	File *asset.File
 }
 
-const (
-	// TODO - add support for other architectures
-	archName = "x86_64"
+var (
+	rhcosArch = "x86_64"
```

Comment:
This repo is also used to build OKD (which uses FCOS, not RHCOS), so something like `isoArch` would be more appropriate.